### PR TITLE
Update hll-aggfunction.md

### PR DESCRIPTION
**Pull request by ziham1531991**

### CODE REVIEW

**Comment by shsagir** [comment]
Learn Build status updates of commit _[c6adb63](https://github.com/ziham1531991/dataexplorer-docs/commits/c6adb63f911a0da4e17c88868fe70a0fb227310f)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/query/hll-aggfunction.md](https://github.com/ziham1531991/dataexplorer-docs/blob/patch-3/data-explorer/kusto/query/hll-aggfunction.md) | :warning:Warning | | [Details](#user-content-9be0fb3aecc03baab94bffa0497e49e3-ed508012)

<a id="9be0fb3aecc03baab94bffa0497e49e3-ed508012"></a>
### [data-explorer/kusto/query/hll-aggfunction.md](https://github.com/ziham1531991/dataexplorer-docs/blob/patch-3/data-explorer/kusto/query/hll-aggfunction.md)
 - **Line 25, Column 119**: **[Warning: link-out-of-scope]** `````File 'includes/data-explorer-estimation-accuracy.md' referenced by link '../../includes/data-explorer-estimation-accuracy.md' will not be built because it is not included in build scope.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210250929586837-1467/BuildReport?accessString=18c54b9420bb471e855857e5c0b006b0cf9985b62032c2c2e258d2ee85f3d483).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

File: data-explorer/kusto/query/hll-aggfunction.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Read about the [underlying algorithm (*H*yper*L*og*L*og) and the estimation accu
 | Name | Type | Required | Description |
 |--|--|--|--|
 | *Expr* |  string | &check; | Expression used for the aggregation calculation. |
-| *Accuracy* |   |   | Controls the balance between speed and accuracy. |
+| *Accuracy* |   |   | Controls the balance between speed and accuracy. If unspecified, the default value is `1`. See [Estimation accuracy](../../includes/data-explorer-estimation-accuracy.md) for supported values. |
```
Comment (YaelSchuster) [inline data-explorer/kusto/query/hll-aggfunction.md:25]:
```suggestion
| *Accuracy* |   |   | Controls the balance between speed and accuracy. If unspecified, the default value is `1`. For supported values, see [Estimation accuracy](#estimation-accuracy). |
```

**Comment by shsagir** [comment]
Learn Build status updates of commit _[ca5b1a6](https://github.com/ziham1531991/dataexplorer-docs/commits/ca5b1a6d25b0a6a6e9cdbf11a924baae5e7f15a3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/query/hll-aggfunction.md](https://github.com/ziham1531991/dataexplorer-docs/blob/patch-3/data-explorer/kusto/query/hll-aggfunction.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210250945244960-1467/BuildReport?accessString=27838c85c1f5c4c3dd749472ed6a1891cc0324846f3266d78942aa8cbe5cec6b).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Court72** [comment]
Merging based on approval comment: https://github.com/MicrosoftDocs/dataexplorer-docs/pull/1467#issuecomment-1290281079

**Pull request merged**